xbmc/music/windows/GUIWindowMusicNav.cpp

```diff
@@ -549,14 +549,14 @@ void CGUIWindowMusicNav::GetContextButtons(int itemNumber, CContextButtons &butt
         // 4. specific per album
         buttons.Add(CONTEXT_BUTTON_SET_CONTENT, 20195);
       }
-      if (item->HasMusicInfoTag() && item->GetMusicInfoTag()->GetArtist().size() > 0)
+      if (item->HasMusicInfoTag() && item->GetMusicInfoTag()->GetArtistString().size() > 0)
```

Comment:
It's better to use real free data in low quality to be able to see all impacts of data when reaching the player.

I did some of my tests before with only empty files and .nfo and missed some player bugs.